pkg/structs/structs.go

```diff
@@ -182,3 +201,20 @@ type GenesisInfo struct {
 	GenesisValidatorsRoot string `json:"genesis_validators_root"`
 	GenesisForkVersion    string `json:"genesis_fork_version"`
 }
+
+// / extra
+type HR struct {
```

Comment:
What does HR mean? We should use a more descriptive name

Reply:
It deviated from HeaderRequest. but I think HeaderData would be fine...

Reply:
done